pxr/usdImaging/usdImaging/lightAdapter.cpp

```diff
@@ -71,15 +71,50 @@ UsdImagingLightAdapter::Populate(UsdPrim const& prim,
 {
     index->InsertSprim(HdPrimTypeTokens->light, prim.GetPath(), prim);
     HD_PERF_COUNTER_INCR(UsdImagingTokens->usdPopulatedPrimCount);
+    _RegisterLightCollections(prim);
 
     return prim.GetPath();
 }
 
 void
-UsdImagingLightAdapter::_RemovePrim(SdfPath const& cachePath,
```

Comment:
Did you mean to remove this?  It's still marked overridden in the header, so it's causing build issues.